CHANGES.rst

```diff
@@ -1,12 +1,19 @@
 Changelog
 =========
 
-1.7 (unreleased)
+1.8 (unreleased)
 ----------------
 
 - Nothing changed yet.
 
 
+1.7 (2015-09-24)
```

Comment:
No, you are still working on the 1.7 (unreleased), just replace "- Nothing changed yet." with your changes, we will use zest.releaser to release version 1.7